### PR TITLE
Remove duplicated annotations section in dex chart

### PR DESCRIPTION
**Pull request by kfurgol**
**Description**

Changes proposed in this pull request:

- Removed duplicated annotations section in dex chart

**Related issue(s)**
#2072

### CODE REVIEW

File: resources/istio-kyma-patch/templates/configmap.yaml [hunk 1]
```diff
@@ -77,6 +77,7 @@ data:
   injection-in-namespaces: |
     istio-system disabled
     kube-system disabled
+    kyma-installer disabled
```
Comment (piotrmsc) [inline resources/istio-kyma-patch/templates/configmap.yaml]: kyma-installer is our ns so we can label it there directly

Comment (kfurgol) [reply resources/istio-kyma-patch/templates/configmap.yaml]: done

**Pull request merged**